src/test/java/net/snowflake/client/jdbc/structuredtypes/ResultSetStructuredTypesLatestIT.java

```diff
@@ -184,13 +184,20 @@ private void testMapAllTypes(boolean registerFactory) throws SQLException {
         assertEquals(Double.valueOf(2.2), object.getD(), 0.01);
         assertEquals(BigDecimal.valueOf(3.3), object.getBd());
         assertEquals(
-            Timestamp.valueOf(LocalDateTime.of(2021, 12, 22, 9, 43, 44)), object.getTimestampLtz());
+            LocalDateTime.of(2021, 12, 22, 9, 43, 44)
```

Comment:
As we discussed - is comparing instants the best way? Maybe we can live with the assert in next line?

Reply:
If it is logically correct my proposal is to stay with those version now. I've created separate [jira](https://snowflakecomputing.atlassian.net/browse/SNOW-1324990) to investigate time/date/timestamp deeper.